refresh_test.py

```diff
@@ -78,7 +78,7 @@ def nodetool_refresh(self, node):
 
     def check_timeout(self):
         assert self.monitors.nodes, 'Monitor node should be set, we will try to get metrics from Prometheus server'
-        cmd = 'curl http://%s:9090/api/v1/query_range?query=scylla_storage_proxy_coordinator_read_timeouts&start=%s&end=%s&step=60s' % (
+        cmd = 'curl http://[%s]:9090/api/v1/query_range?query=scylla_storage_proxy_coordinator_read_timeouts&start=%s&end=%s&step=60s' % (
```

Comment:
this will breake IPv4

Reply:
we tested it, it won't break IPv4

Reply:
We ran such http command with ipv4 - it worked